docs/src/AlgebraicGeometry/Schemes/intro.md

```diff
@@ -0,0 +1,33 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["intro.md"]
+```
+
+# Introduction
+
+## Content
+
+The schemes part of OSCAR comprises algorithms addressing affine schemes. More functionality
+(e.g. for general covered schemes and toric schemes) exists in experimental state.
+
+
+## Status
+
+This project is work-in-progress.
+
+
+## Contact
+
+Please direct questions about this part of OSCAR to the following people:
+* [Matthias Zach](https://www.mathematik.uni-kl.de/en/agag/people/members/seite),
+* [Anne Frühbis-Krüger](https://uol.de/anne-fruehbis-krueger),
+* [Simon Brandhorst](https://www.math.uni-sb.de/ag/brandhorst/index.php?lang=en).
+
+You can ask questions in the [OSCAR Slack](https://join.slack.com/t/oscar-system/shared_invite/zt-thtcv97k-2678bKQ~RpR~5gZszDcISw). Alternatively, you can [raise an issue](https://github.com/oscar-system/Oscar.jl/issues/new/choose) on github.
```

Comment:
Please split this into two lines.

Also note that the OSCAR invite URL can sometimes change. So I'd rather not put it into too many places. Perhaps we can have one central place this is mentioned, and then link to that. Perhaps that central place could even be <https://oscar.computeralgebra.de/community/#slack> or so.

Reply:
Actually perhaps we can setup an URL (say https://slack.oscar-system.org/invite or https://oscar-system.org/slack/invite) which redirects to the actual invite URL. Then we can use the fixed URL everywhere and only need to update one place when it changes

Reply:
I followed your first suggestion and linked to https://oscar.computeralgebra.de/community/#slack for details on reaching us via slack.

In the same spirit, I have linked to https://oscar.computeralgebra.de/community/#how-to-report-issues for details on raising an issue, because it list more details there. (You need a github account, issue could also be raised in GAP.jl, Polymake.jl, Singular.jl, Polymake.jl but we move it if needed... etc.)

In both cases, we only need to modify data on the OSCAR website. This should make it a lot easier to maintain.